### PR TITLE
Pool overview page redesign

### PR DESCRIPTION
**Pull request by kattylucy**
### Description

- Redesign menu on the left and primary/secondary colors for the buttons
- Redesign layout and router to use outlet 
- Check on storybook components
- Delete alternative color theme (dark / light ) 

#2390

### Approvals

- [ ] Dev

### CODE REVIEW

**Review by sophialittlejohn — COMMENTED** [review]
Nice work @kattylucy 🔥 

The Banner that's sometimes displayed in the bottom right corner could most definitely benefit from a style upgrade. What do you think about a black background with white text? 

In general do you think it would make sense to replace the primary color in the theme for gold to ease the transition?

File: fabric/src/theme/tokens/theme.ts [hunk 1]
```diff
@@ -29,6 +34,7 @@ const lightColors = {
   backgroundInput: 'white',
   backgroundThumbnail: grayScale[500],
   backgroundInverted: grayScale[900],
+  backgroundBlack: black,
```
Comment (onnovisser) [inline fabric/src/theme/tokens/theme.ts:37]: I would change the `backgroundInverted` to be `black` and use that for the sidebar

Comment (onnovisser) [reply fabric/src/theme/tokens/theme.ts:37]: or better yet, change grayScale[900] to be `#252B34`.

File: fabric/src/theme/tokens/theme.ts [hunk 2]
```diff
@@ -14,11 +14,16 @@ const statusWarningBg = yellowScale[50]
...
+const black = '#252B34'
+const gold = '#FFC500'
+
+const colors = {
   textPrimary: grayScale[900],
   textSecondary: grayScale[800],
   textDisabled: grayScale[500],
   textInverted: 'white',
+  textGold: gold,
+  textBlack: black,
```
Comment (onnovisser) [inline fabric/src/theme/tokens/theme.ts:26]: I would just use textPrimary. If the primary text color has change, we should update that instead

File: centrifuge-app/src/utils/useListedPools.ts [hunk 3]
```diff
@@ -40,3 +48,83 @@ export function useListedPools() {
...
+  const flattenedData =
+    data?.pools?.nodes.flatMap((pool: any) =>
+      pool.poolSnapshots.nodes.map((snapshot: any) => ({
+        netAssetValue: snapshot.netAssetValue,
+        decimals: pool.currency.decimals,
+      }))
+    ) || []
+
+  // Aggregate NAV from last year
+  const aggregatedNetAssetValue = flattenedData.reduce((accumulator: any, item: FlattenedDataItem) => {
```
[the remark this answers is not in the document]
Comment (sophialittlejohn) [reply centrifuge-app/src/utils/useListedPools.ts:112]: you'll have to import it 
```js
import Decimal from 'decimal.js-light'
```

[the remark this answers is not in the document]
Comment (kattylucy) [reply centrifuge-app/src/utils/useListedPools.ts:112]: Oh i missed this change. THANK YOU!

**Pull request merged**